libcloud/compute/drivers/dimensiondata.py

```diff
@@ -3830,6 +3830,66 @@ def ex_create_node_uncustomized(self,
 
         Create Node in MCP2 Data Center
 
+        >>> from pprint import pprint
```

Comment:
Agree. I've moved created a folder for dd in docs/examples. Sample code moved to folder.